kyuubi-server/web-ui/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "kyuubi-ui",
+  "private": true,
+  "version": "1.6.0-SNAPSHOT",
```

Comment:
please rebase master and change it to 1.7.0-SNAPSHOT

Reply:
sure.